### PR TITLE
[FIX] Sandbox proxy failing

### PR DESCRIPTION
**Pull request by MiguelCarranza**

### CODE REVIEW

**Comment by jeiting** [comment]
@MiguelCarranza Do you mind elaborating a bit on what happened? I looked up the setting you changed but I'm not sure why Apple suddenly started breaking.

Also, heads up this is a public repo.

**Comment by MiguelCarranza** [comment]
Yes. I am not super familiar with this library, so there might be a better way to do it.

We noticed that the receipt checker on the website was broken last weekend. After investigating how the website does it, I noticed that it goes to Sandbox first, and if Apple returns a `21008`, we forward to production. 

That has always worked, so there was no change of behavior or code on our side.

I tested this code with a production receipt, and it worked. Sending a sandbox receipt, it would return a `21007` (Token sent to production, but it's sandbox) as expected. 

However, when posting to `/sandbox`, the proxy to `https://sandbox.itunes.apple.com/verifyReceipt` would return this:
Screen Shot 2020-04-20 at 8 17 02 AM

I assumed Apple could have changed something in sandbox that was not compatible with our proxy. I looked at [http-proxy docs](https://www.npmjs.com/package/http-proxy) and tested several things.  The `changeOrigin` option made the trick.

**TL;DR**: Production proxy was working fine, but sandbox was not. `changeOrigin` fixes it for sandbox, and it does not seem to be required for production (yet).
